src/uct/api/v2/uct_v2.h

```diff
@@ -480,9 +497,22 @@ typedef struct uct_iface_is_reachable_params {
  */
 typedef struct uct_ep_connect_to_ep_params {
     /**
-     * Reserved, must be 0.
+     * Mask of valid fields in this structure and operation flags, using
+     * bits from @ref uct_ep_connect_to_ep_param_field_t. Fields not specified
+     * in this mask will be ignored. Provides ABI compatibility with respect to
+     * adding new fields.
      */
     uint64_t                      field_mask;
+
+    /**
+     * Device address length
+     */
+    size_t                        device_addr_length;
+
+    /**
+     * Endpoint address length
```

Comment:
pls document what happens if length is not provided

Reply:
added comment